test/test-scripts/helpers/topUpUsersVoucher.js

```diff
@@ -0,0 +1,60 @@
+import { Contract, ZeroAddress } from "ethers";
+import {
+  VOUCHER_HUB_ADDRESS,
+  provider,
+  setBalance,
+  getVoucherManagerWallet,
+} from "./utils.js";
+import { voucherHubAbi } from "../abis/VoucherHub.js";
+
+export const topUpUsersVoucher = async ({ ownerAddress, value = 1000 }) => {
+  if (!ownerAddress) {
+    throw Error("missing ownerAddress");
+  }
+
+  console.log(`Topping-up ${value} on user ${ownerAddress} voucher`);
+
+  const voucherHubContract = new Contract(
+    VOUCHER_HUB_ADDRESS,
+    voucherHubAbi,
+    provider
+  );
+
+  const voucherAddress = await voucherHubContract.getVoucher(ownerAddress);
+
+  if (voucherAddress === ZeroAddress) {
+    throw Error(`${ownerAddress} has no voucher, cannot top-up`);
+  }
+
```

Comment:
pour une meilleur lisibilité peut etre ajouter une constante :) (optionnel) 

```suggestion
const NANO_RLC_CONVERSION = 10n ** 9n;

```